### PR TITLE
Fix some wrong comments in tracker.hpp

### PR DESCRIPTION
**Pull request by klchang**
Fix some wrong comments in `modules/tracking/include/opencv2/tracking/tracker.hpp`.

### CODE REVIEW

File: modules/tracking/include/opencv2/tracking/tracker.hpp [hunk 1]
```diff
@@ -1227,12 +1227,12 @@ class CV_EXPORTS_W TrackerKCF : public Tracker
...
 
     /**
-    * \brief Read parameters from file, currently unused
+    * \brief Read parameters from file
     */
     void read(const FileNode& /*fn*/);
 
     /**
-    * \brief Read parameters from file, currently unused
+    * \brief Write parameters in a file
```
Comment (terfendail) [inline modules/tracking/include/opencv2/tracking/tracker.hpp]: IMHO here should be `Write parameters to a file`

Comment (klchang) [reply modules/tracking/include/opencv2/tracking/tracker.hpp]: IMHO, the principles to fix these wrong comments is as few changes as possible, at the same time, consistent with existed comments. In the line 1116 of tracker.hpp, there is a comment, i.e. `* \brief Write parameters in a file`, so I copyed it. If only from the view of its meaning, 'Write parameters to a file' should be better.

**Comment by terfendail** [comment]
@klchang Thank you for contribution.
The change looks fine for me now, however it couldn't be tested properly due to an issue with existing CI infrastructure that makes impossible to test changes proposed from master branch.
Could you please move the change to other branch?

**Comment by klchang** [comment]
@terfendail OK. I have moved the changes to a new branch in my repository. Thank you for your attention.

**Pull request merged**